vamb/__main__.py

```diff
@@ -1195,17 +1194,26 @@ def cluster_and_write_files(
         cuda=cuda,
         rng_seed=seed,
     )
-    # This also works correctly when max_clusters is None
+
     clusters = itertools.islice(cluster_generator, cluster_options.max_clusters)
-    cluster_dict: dict[str, set[str]] = dict()
 
     # Write the cluster metadata to file
-    with open(Path(base_clusters_name + "_metadata.tsv"), "w") as file:
+    with open(Path(base_clusters_name + "_metadata.tsv"), "a") as file:
```

Comment:
This 'a' is incorrect, because the file has a header. So, if the file already contain content, the header will be somewhere not at the top, making it an invalid TSV file.